packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -357,37 +357,46 @@ function WorklistDataGrid(props) {
     }
   };
 
-  const selectMultipleRows = useCallback((newEndRowIndex, rangeInfo) => {
-    if (!onRowSelect || !rangeInfo) {
-      return;
+  const getRowsInRangeToUnselect = useCallback((startOfRange, prevEndOfRange, newEndOfRange) => {
```

Comment:
Is the startOfRange the anchor or is the prevEndOfRange the anchor.  I also don't think we should abbreviate names when possible.  We should spell out "previous".  It would be helpful to have documentation for this method.

Reply:
The start of the range is the anchor. Since anchor is not a globally defined term, we went with startOfRange. It is best to explain with an example:

- User clicks row 4 (This establishes the anchor or the starting point for range selection. As there is no range as yet, there is no end of range)
- User Shift+Click row 6 (This means we now have a range of selected rows. Since no range existed before the previousEnd is not defined. For the newly established range, the end is row 6)
- User Shift+Click row 8 (This means that the range is being extended from it previous end at row 6 to the new end at row 8. That is, the range is now from row 4 to row 8)